code/_helpers/maths.dm

```diff
@@ -0,0 +1,11 @@
+//from bay wth circuitry, arfs change
+
+#define ceil(x) (-round(-(x)))
+
+/proc/ToDegrees(radians)
+	// 180 / Pi ~ 57.2957795
+	return radians * 57.2957795
+
+/proc/ToRadians(degrees)
+	// Pi / 180 ~ 0.0174532925
+	return degrees * 0.0174532925
```

Comment:
Already in math DEFINE

Reply:
wheeeere?

Reply:
ceil is not the same.

Reply:
Yes they are.
ceil(5.7887) -> 6
CEILING(5.7887, 1) -> 6
![image](https://user-images.githubusercontent.com/44811257/88612592-29c62000-d051-11ea-85da-a138da847565.png)

Reply:
fixing

Reply:
having issue and I don't know whyyyy

            for(var/i = 1 to CEILING(length(assembly_components) / components_per_page), 1)
code\modules\integrated_electronics_arfs\core\assemblies.dm:202:error: incorrect number of macro arguments